File: RSMK2/RSMK2/StaffOperationImpl.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace RSMK2
{
    public class StaffOperationImpl : MarshalByRefObject, StaffOperation
    {
        public static List<Staff> lst = new List<Staff>();

        public List<Staff> GetAll() => lst;

        public List<Staff> Add(Staff newItem)
        {
            lst.Add(newItem);
            return lst;
        }

        public List<Staff> DeleteAt(int index)
        {
            lst.RemoveAt(index);
            return lst;
        }

        public double CountTotal()
        {
            double result = 0d;
            foreach (var item in lst)
            {
                result += item.GetSalary();
            }
            return result;
        }
    }
}

File: zwsserver/src/operation/StaffOperationImpl.java
package operation;

import types.Staff;
import java.util.ArrayList;
import java.util.List;

public class StaffOperationImpl implements StaffOperation {

    public static List<Staff> lst = new ArrayList<Staff>();

    static {
        lst.add(new Staff("Mark Twen", 255, "admin", 1500));
        lst.add(new Staff("Doctor Who", 32767, "CEO", 7500d));
    }

    @Override
    public List<Staff> GetAll()
    {
        return lst;
    }

    @Override
    public List<Staff> Add(Staff newItem)
    {
        lst.add(newItem);
        return lst;
    }

    @Override
    public List<Staff> DeleteAt(int index)
    {
        lst.remove(index);
        return lst;
    }

    @Override
    public double CountTotal()
    {
        double result = 0;
        for (var item : lst)
        {
            result += item.getSalary();
        }
        return result;
    }
}
